parso/python/tokenize.py

```diff
@@ -143,13 +143,13 @@ def _create_token_collection(version_info):
         Floatnumber = group(Pointfloat, Expfloat)
         Imagnumber = group(r'[0-9](?:_?[0-9])*[jJ]', Floatnumber + r'[jJ]')
     else:
-        Hexnumber = r'0[xX][0-9a-fA-F]+'
-        Binnumber = r'0[bB][01]+'
+        Hexnumber = r'0[xX][0-9a-fA-F]+[lL]?'
+        Binnumber = r'0[bB][01]+[lL]?'
         if version_info >= (3, 0):
-            Octnumber = r'0[oO][0-7]+'
+            Octnumber = r'0[oO][0-7]+[lL]?'
         else:
-            Octnumber = '0[oO]?[0-7]+'
-        Decnumber = r'(?:0+|[1-9][0-9]*)'
+            Octnumber = '0[oO]?[0-7]+[lL]?'
+        Decnumber = r'(?:0+[lL]?|[1-9][0-9]*[lL]?)'
         Intnumber = group(Hexnumber, Binnumber, Octnumber, Decnumber)
```

Comment:
I feel like the repetition of `[lL]?` is better written as:

```
Intnumber = group(Hexnumber, Binnumber, Octnumber, Decnumber)
if version_info[0] < 3:
    Intnumber += '[lL]?'
```

Reply:
thanks for the code review :+1: